### PR TITLE
chore(select): Fix JS example to actually work

### PR DESCRIPTION
**Pull request by williamernest**
Refs: #2275 
Doesn't fix the entire README, which needs to be redone, but it does fix our JS example to actually run until we get to it.

### CODE REVIEW

File: packages/mdc-select/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Then with JS
 ```js
 import {MDCSelect} from '@material/select';
```
Comment (moog16) [inline packages/mdc-select/README.md]: should we also then remove this?

Comment (moog16) [inline packages/mdc-select/README.md]: should we also then remove this?

Comment (williamernest) [reply packages/mdc-select/README.md]: Yes, but this readme doesn't ever reference the `Importing the JS component` section, so I'll add that under this code block.

Comment (williamernest) [reply packages/mdc-select/README.md]: Yes, but this readme doesn't ever reference the Importing the JS component section, so I'll add that under this code block.

File: packages/mdc-floating-label/mdc-floating-label.scss [hunk 2]
```diff
@@ -29,6 +29,7 @@
   transform-origin: left top;
   transition: mdc-floating-label-transition(transform), mdc-floating-label-transition(color);
   cursor: text;
+  will-change: transform;
```
Comment (moog16) [inline packages/mdc-floating-label/mdc-floating-label.scss]: does this fix the jitter?

**Comment by kfranqueiro** [comment]
This seems reasonable given that we have other READMEs that do the same, but we currently have inconsistency in whether we reference the globals or use syntax that assumes ES6 imports. We should standardize on one or the other eventually.

**Pull request merged**